hacep/src/main/java/org/kie/hacep/core/infra/consumer/DefaultKafkaConsumer.java

```diff
@@ -111,72 +112,79 @@ private void restartConsumer() {
     @Override
     public void stop() {
         stopConsume();
-        stopPollingControl();
-        stopPollingEvents();
+        pollNothing();
```

Comment:
Is this method necesarry if it does nothing? Or please provide better name which explains what it does and not only what it does not.

Reply:
It's not necessary anymore, removed